README.md

```diff
@@ -74,18 +74,43 @@ http://localhost:9790/public/api/swagger-ui.html
 http://localhost:9790/public/api/docs
 ```
 
+## Running as Docker Mysql container
+Before running the project, you need to run the following command to start the mysql container:
+
+```
+docker compose up -d --build mysql
+```
+
+If you want to recreate the mysql container, you can run the following command:
+```
+docker compose up --force-recreate -d --build mysql
+```
+
+If you want to stop the mysql container, you can run the following command:
+```
+docker compose down -v mysql
+```
+
 ## Running as Docker container
 
-After building the project, run below to build and tag the docker image.
+Before running the project, you need to run the following command to start the project container:
 
 ```
-docker build -t ays/ays-be .
+docker compose up -d --build mysql
 ```
 
-For running the docker image, run the following command:
+```
+docker compose up -d --build ays-be
+```
+
+If you want to recreate the project container, you can run the following command:
+```
+docker compose up --force-recreate -d --build 
+```
 
+If you want to stop the project container, you can run the following command:
 ```
-docker run -p 9790:9790 ays/ays-be
+docker compose down -v 
 ```
 
 ## Running with Docker Compose
```

Comment:
@skayikci What do you think about removing everything under this heading since it is no longer necessary for the local environment?

Reply:
@agitrubard Can we add some information to show MySQL through bash.

Here are all commands to show it

- docker exec -it ays-db bash
- mysql -u ays -p
- Enter password
- USE ays
- show (table_name)